### ISSUE TITLE
BinanceWebSocketApiManager does not accept parameter exchange="binance.com-coin-futures"

### ISSUE THREAD

**Issue by MichitakaTsuda** [oliver-zehentleitner/unicorn-binance-websocket-api#170]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x] Bug
- [ ] Feature Request
- [x] Technical Help
- [ ] Other

#### Operating System? (include version)
- [x] macOS
- [ ] Windows
- [ ] Linux (include flavour)

#### Which endpoint do you connect?
```
binance.com-coin-futures
```

#### Python Version Requirement
- [x] I am using Python 3.6.1 or above

#### Exact Python Version?
```
Python 3.8.2
```

#### UNICORN Binance WebSocket API Version?
```
Latest version as of today
```

#### Description of your issue
```
As mentioned in title, BinanceWebSocketApiManager does not accept parameter exchange="binance.com-coin-futures".
The reason is unicorn_binance_websocket_api_manager.py does not define binance.com-coin-futures in if-clause.

Since the documentation describes 'binance.com-coin-futures' is applicable for exchange parameter, I thought that this endpoint has mistakenly deleted or deprecated somehow, while change log does not mention anything about this.
Is there any way to connect wss://dstream.binance.com/ endpoint? 

Waiting for your catch ups.
Sincerely,
```

**Comment by oliver-zehentleitner** [comment]
Hi!
coin-futures is not released yet and only available in the dev stage: https://github.com/oliver-zehentleitner/unicorn-binance-websocket-api/blob/master/CHANGELOG.md

If you want to try the dev stage just install this way: https://github.com/oliver-zehentleitner/unicorn-binance-websocket-api#from-the-latest-source-dev-stage-with-pip-from-github

**Comment by MichitakaTsuda** [comment]
Appreciate for your immediate response!
Sorry I didn't fully read through dev version which made me confused! My bad.

By the way, I propose that the documentation which is suggested during handling Exceptions should refer somewhat to the exchange parameter "coin-futures" is under dev stage, which helps users' clearer understanding!

Like: `exchange (str) – Select binance.com, binance.com-testnet, binance.com-margin, binance.com-margin-testnet, binance.com-isolated_margin, binance.com-isolated_margin-testnet, binance.com-futures, binance.com-futures-testnet, binance.com-coin-futures*, binance.je, binance.us, trbinance.com, jex.com, binance.org or binance.org-testnet (default: binance.com) *only available on dev-stage`

Source: https://oliver-zehentleitner.github.io/unicorn-binance-websocket-api/unicorn_binance_websocket_api.html#module-unicorn_binance_websocket_api.unicorn_binance_websocket_api_manager

**Comment by oliver-zehentleitner** [comment]
I think it was a mistake that i updated the docs... sorry for confusion!

On pypi you see the description related to the version: https://pypi.org/project/unicorn-binance-websocket-api/1.30.0/

We added coin-futures and it should be released with 1.31.0 and prepared README and other things, but docs should have been excepted.

This will be cleaned with the next release ;)
